### ISSUE TITLE
Accessibility-Ready Review: Initial Review

### ISSUE THREAD

**Issue by davidakennedy** [WordPress/twentyseventeen#288]
I completed an initial accessibility-ready review for the theme. Here's what I found:

**Keyboard Navigation**

Pass: No issues, overall.
- Recommended: Should the menu toggle have a solid border on `:focus`? Right now, it's dotted, like links.

**Controls**

Pass: Pass: No issues, overall.
- Recommended: The button, `<button class="menu-scroll-down"><span class="screen-reader-text">Scroll Down</span></button>` really performs the task of a link, and should probably be changed to an `<a>`.

**Skip Links**

Pass: No issues.

**Forms**

Pass: No issues.

**Headings**

Needs work:
- No `<h1>` exist on the front page when the panel solution is set up. This also happens when the front page is a static page and not posts.

**ARIA Landmarks**

Pass: No issues.

**Link Text**

Needs work:

`link-more` links on search pages need post title to have more context.

**Contrasts**

Needs work:
- `.site-description`: #777 on `#fafafa`. @melchoyce says this can change to `#666666`.
- `.social-navigation` links: `#ffffff` on `#bbb`. @melchoyce says this will change to `#767676` with `:hover` and `:focus` moving to `#333333`.

**Images**

Pass: No issues. Will need to retest when SVGs merged.

**Media**

Pass: No issues.

**Screen Reader Text**

Pass: No issues.

**Disallowed**:
- Any positive tabindex attribute. Negative or zero value tabindex is allowed in specific circumstances (assessed on a case-by-case basis): Pass.
- The inclusion of the accesskey attribute: Pass.
- Spawning new windows or tabs without warning the user: Pass.

**Recommended**:
- Enlarging text: Pass.
- title attributes: Remove title attribute on search form.

**Comment by davidakennedy** [comment]
@laurelfulford One small thing I noticed while testing something else:

This should have screen reader text: https://github.com/WordPress/twentyseventeen/blob/master/inc/template-tags.php#L40

Like this: https://github.com/WordPress/twentyseventeen/blob/master/inc/template-tags.php#L22

For consistency sake.

**Comment by laurelfulford** [comment]
There should be a separate PRs for each of the reported issues and recommendations. 🙂

**Comment by davidakennedy** [comment]
These should all be addressed now.

**Comment by rianrietveld** [comment]
My 2 cents on WordCamp Netherlands:
I love the theme, beautiful!

Note: I did not read all the replies with this issue, so maybe some issues are already addresses (sorry for that).


Links:
- outside entry-content there is no hover style on links
- the links in comments have no underline nor style on hover
- menu top level no style on hover

About the entry-meta in an archive, for example:

```
	<span class="posted-on">
		<span class="screen-reader-text">Posted on</span> 
		<a href="http://trunk.dev/2016/09/27/hello-world/" rel="bookmark">
			<time class="entry-date published" datetime="2016-09-27T17:06:23+00:00">September 27, 2016</time>
			<time class="updated" datetime="2016-10-12T11:38:03+00:00">October 12, 2016</time>
		</a>
	</span>
	<span class="byline"> by <span class="author vcard">
		<a class="url fn n" href="http://trunk.dev/author/webbeheer/">webbeheer</a>
	</span>
</span>
```

There is a link with as text the date of a post. This links goes to the post itself, and that link is repeated on the text for the post type. The date is a link to a post, and below that there is also a link to the same post.
On the single page this date is also a link to the age itself
IMHO this date text should not be a link at all, not on the archive and not on the single page.

After that is a link with the name of the author, that goes to the archive of the author.
Including the text "by" to the link is giving the link text more meaning/context .

Also if there are comments the time/date is the comments are also just a link to the date but in fact a link to the comment. Adding more context in the link text might be helpful to screen reader users.

Default header colour (minor issue, but it triggers a validation error)
The site title = #fff
the background = #fafafa
if images are disabled (or still loading) the site-title is not visible
the heading background can be adjusted by the user, but maybe it should be ok out of the box?

Search form: 
Remove the title attribute from the input field

Pagination:
Add screen reader text or aria-label to the links to the pages to give more context to the link 
for example:  1 could be Page 1 (page hidden by screen reader text
and also indicate the active page
And add the text _Page_ to the next page arrow link

I think that the scroll down button should stay a button (disagree with https://github.com/laurelfulford/twentyseventeen/commit/bcd5af4d22aedafd97a80be1c97d8de14e132ca7)
the "link" doesn't go to a href, it's just an action to scroll the page down a but, so it should stay a button.

I hope this is helpful
